### PR TITLE
Update to 1.20.4

### PR DESCRIPTION
**Pull request by mms0316**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -8,15 +8,15 @@ description=Support for litematica on client to worldedit on server communicatio
 modVersion=1.0.0
 
 # Fabric
-minecraftVersion=1.20.1
-loaderVersion=0.14.21
-yarnMappings=1.20.1+build.10
+minecraftVersion=1.20.4
+loaderVersion=0.15.11
```
Comment (dlee13) [inline gradle.properties]: Revert the loader version change. We don't need to bump this.

File: gradle.properties [hunk 2]
```diff
@@ -8,15 +8,15 @@ description=Support for litematica on client to worldedit on server communicatio
...
 
 # Paper
 apiVersion=1.20
 
 # Dependencies
-fabricVersion=0.83.1+1.20.1
-paperVersion=1.20.1-R0.1-SNAPSHOT
+fabricVersion=0.97.1+1.20.4
+paperVersion=1.20.4-R0.1-SNAPSHOT
 litematicaFileId=4593296
```
Comment (dlee13) [inline gradle.properties]: Update the litematica file id. Latest build is here:
https://www.curseforge.com/minecraft/mc-mods/litematica/files/5393557
So the new file id that we depend on is `5393557`

File: gradle.properties (hunk 1 above)
Comment (mms0316) [reply gradle.properties]: Looking at https://github.com/FabricMC/fabric-loader/releases?page=2, it's not 100% clear when fabric loader started supporting 1.20.4, so, which is preferred?

- Keep at 0.14.21, which didn't support 1.20.2+
- Move to 0.14.23 which supported 1.20.3. IIRC 1.20.3 and 1.20.4 were mostly compatible with each other
- Move to 0.15.2 which is before 1.20.5 support

File: gradle.properties (hunk 2 above)
Comment (mms0316) [reply gradle.properties]: However, shouldn't it be updated to the first Litematica version for 1.20.4?
https://www.curseforge.com/minecraft/mc-mods/litematica/files/4946471

File: gradle.properties (hunk 1 above)
Comment (dlee13) [reply gradle.properties]: 0.14.23

File: gradle.properties (hunk 2 above)
Comment (dlee13) [reply gradle.properties]: Our fabric.mod.json has dependency on any version of litematica. So this one is only to make sure that the litematica internals that we're using are still there. Generally, we would want users to use the latest.

**Pull request merged**